### DISCUSSION TITLE
How do reported bugs get priority?

### DISCUSSION THREAD

**Discussion by mona-mk** [apache/superset#16270] [General]
I'm wondering how can one increase the priority of a bug, or at least get some rough timeline for a fix? e.g. this bug is troublesome for us: https://github.com/apache/superset/issues/15738

**Comment by villebro** [comment]
I is usually very helpful to create an example that reproduces the problem. I'll add comments to the original ticket, thanks for raising.

**Reply by mona-mk** [reply]
Thanks a lot Ville. It feels like a relief seeing the messages are being read by Superset guys :) Noticed your comment, will continue there.
